urllib3/util/ssl_.py

```diff
@@ -8,6 +8,13 @@
 HAS_SNI = False
 create_default_context = None
 
+# Maps the length of a digest to a possible hash function producing this digest
+HASHFUNC_MAP = {
+    16: md5,
+    20: sha1,
+    32: sha256,
+}
+
```

Comment:
This is used as a constant, so capitalized and moved from being defined inside the function to module scope.